rootfs/api/models/app.py

```diff
@@ -458,7 +459,7 @@ def _default_scale(self, user, release):
         else:
             structure = {'web': 1}
 
-        self.scale(user, structure)
+        self._deploy_app(structure, release, [])
```

Comment:
Making a real distinction between scale and deploy since this is within the deploy operation